package.json

```diff
@@ -14,17 +14,17 @@
   },
   "dependencies": {
     "eslint-plugin-es": "^4.1.0",
-    "eslint-utils": "^2.0.0",
+    "eslint-utils": "^3.0.0",
     "ignore": "^5.1.1",
     "is-core-module": "^2.3.0",
     "minimatch": "^3.0.4",
     "resolve": "^1.10.1",
     "semver": "^6.1.0"
   },
   "devDependencies": {
-    "@mysticatea/eslint-plugin": "^10.0.3",
+    "@mysticatea/eslint-plugin": "^13.0.0",
     "codecov": "^3.3.0",
-    "eslint": "^6.3.0",
+    "eslint": "^8.0.0-0",
```

Comment:
Sticking to ESLint v7 in the dev dependencies would unblock this PR if you add checks to make sure everything works with this ESLint version. Upgrading ESLint here is just for the local environment used to develop the project, that must not block this PR to add ESLint v8 compatibility to the package.